lib/model/model.go

```diff
@@ -626,6 +622,11 @@ func (m *Model) Completion(device protocol.DeviceID, folder string) FolderComple
 			return true
 		}
 
+		if file, ok := rf.Get(device, ft.Name); ok && file.Invalid {
```

Comment:
This, while being the point of the exercise, is scary. It makes completion calculations at least twice as expensive as they already were. Do we understand the impact here?

Reply:
Argh this entire thing shouldn't be here, this is legacy of my first proof-of-concept. The entire point of adding invalid to the global version list was to get rid of these additional lookups by excluding invalid files in the db iteration if requested. So I will have to test again whether my entire scheme actually works.

Reply:
Hurrey, it is working without it and now it is blazing fast. I was worried why it took so long to get idle before, now I know why - thanks for catching this!